database/seeders/EventsGerolsteinSeeder.php

```diff
@@ -130,22 +137,22 @@ public function runSamstagabendGruppenphase(): void
     }
 
     /**
-     * Run the "Tanzen, Sketch & Gesang" event seeds.
+     * Run the "Tanzen & Gesang" event seeds.
      */
-    public function runSamstagabendTanzenSketchGesang(): void
+    public function runSamstagabendTanzenGesang(): void
     {
-        // check if event with name "Tanzen, Sketch & Gesang" exists
-        $event = Event::where('name', 'Tanzen, Sketch & Gesang')->first();
+        // check if event with name "Tanzen & Gesang" exists
+        $event = Event::where('name', 'Tanzen, Gesang')->first();
```

Comment:
You check if the Event "Tanzen, Gesang" exists but the event name a few lines later is "Tanzen & Gesang"